charts/redpanda/values.schema.json

```diff
@@ -961,7 +961,6 @@
             },
             "external": {
               "type": "object",
-              "minProperties": 1,
```

Comment:
why can this be removed?

Reply:
This removes the need for a user to have a populated `external` section when defining a schemaRegistry listener. I included this change because I wanted the ability to comment out (or not include) the listener's external configuration, and instead have the templates rely on the default listener values (and the external section) to determine how to configure this listener for external access. This works for because both the LoadBalancer template defaults to values in the following order:
1. `listeners.*.external.*.advertisedPorts[0]`
1. `listeners.*.external.*.port`
1. `listeners.*.port`

https://github.com/redpanda-data/helm-charts/pull/320/files#r1113645104

The other template that uses this section is the Nodeport service, and it default to values in the following order:
1. `listeners.*.external.*.advertisedPorts[0]`
1. `listeners.*.port`

https://github.com/redpanda-data/helm-charts/blob/main/charts/redpanda/templates/services.nodeport.yaml#L66-L67

But a good question would be: why is this included in this PR? I included it just because I was wanting to test the above behavior and the schema wouldn't let me. But it doesn't necessarily need to be in this PR. Another reason for not including it would be that there are lots of changes like this that could be made throughout to ensure users can easily use a subset `values.yaml` file without running into issues.

Reply:
I understand. Yes, let's please do this as a separate PR and as you say, address the usability of this section in general. Would you mind creating an issue so we don't lose the idea?

Reply:
Here is the issue tracking this https://github.com/redpanda-data/helm-charts/issues/359